tests/containers/bci_prepare.pm

```diff
@@ -33,6 +33,9 @@ sub packages_to_install {
     my $arch = get_required_var('ARCH');
     my $scc_timeout = 1200;    # SCC can take really long timetimes
 
+    # Avoid PackageKit to conflict about lock with zypper
+    script_run("pkcon quit");
```

Comment:
Normally, `script_run` shall be used with `die_on_timeout`.
https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L972

```suggestion
    script_run("pkcon quit", die_on_timeout => 0);
```